libc/config/darwin/x86_64/entrypoints.txt

```diff
@@ -119,6 +119,10 @@ set(TARGET_LIBM_ENTRYPOINTS
     #libc.src.math.ceill
     #libc.src.math.coshf
     #libc.src.math.cosf
+    #libc.src.math.daddl
+    #libc.src.math.daddf128
+    #libc.src.math.ddivl
+    #libc.src.math.ddivf128
```

Comment:
There were previously no `float128` entrypoints in this file either so there might be an issue with `float128` on this target too.